CHANGES.rst

```diff
@@ -418,6 +418,8 @@ associations
 
 - Fixed treating non-science as TSO [#3601]
 
+- Added the fxd_slit keyword as the third optical component [#3607]
```

Comment:
This new entry needs to be up at the top of this file in the new 0.13.4 section.